ras_backstage/exception/exceptions.py

```diff
@@ -0,0 +1,16 @@
+
+class RasError(Exception):
+    """
+    RAS domain-specific errors. The ctor accepts either a string describing the error, or a list of strings
```

Comment:
Is this taken from common? Apparently it's short for constructor. I would just put the comment closer to the init method.

Reply:
Yes it is